Functions.Templates/Templates/SqlInputBinding-CSharp-Isolated/.template.config/template.json

```diff
@@ -47,7 +47,7 @@
           "ManualInstructions": [],
           "args": {
             "referenceType": "package",
-            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Sql", "version": "1.0.122-preview",
+            "reference": "Microsoft.Azure.Functions.Worker.Extensions.Sql", "version": "3.0.143-preview",
```

Comment:
So we do need to get a GA version of the worker out then - right @MaddyDev ?

Reply:
yes